src/pool/srv_pool_map.c

```diff
@@ -148,6 +165,22 @@ update_one_tgt(struct pool_map *map, struct pool_target *target,
 				DP_TARGET(target));
 			return -DER_INVAL;
 		case PO_COMP_ST_UP:
+			D_DEBUG(DF_DSMS, "change "DF_TARGET" to UPIN %p\n",
+				DP_TARGET(target), map);
+			/*
+			 * Need to update this target AND all of its parents
+			 * domains from NEW -> UPIN
+			 */
+
+			target->ta_comp.co_flags &= ~PO_COMPF_DOWN2OUT;
```

Comment:
any reason why can not use same handling as "PO_COMP_ST_NEW"? 
PO_COMPF_DOWN2OUT flag seems should not exist if the component is in PO_COMP_ST_UP status.